janusgraph-test/src/test/java/org/janusgraph/diskstorage/configuration/builder/ReadConfigurationBuilderTest.java

```diff
@@ -164,6 +165,20 @@ public void shouldSetupBackendPreferenceTimestampProvider() {
         verify(globalWrite).set(TIMESTAMP_PROVIDER, timestampProviders);
     }
 
+    @Test
+    public void shouldSetupBackendPreferenceTimestampProviderBeforeAnythingElse() {
```

Comment:
Maybe an admin can disable the Codacy check here. We should probably even disable it globally, because there are lots of cases where a test is perfectly fine without `assert()` or `fail()`.

Reply:
Disabled this check

Reply:
This check is disabled now in Codacy but it still complains about this issue. I guess, it will be resolved later. Nevertheless, Codacy never blocks a PR from being merged.